internal/cli/cli.go

```diff
@@ -103,13 +103,34 @@ func parseCmd() *cobra.Command {
 		},
 	}
 
+	var documentGetId string
+	var documentGetShareId string
+	documentGetCmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get a document",
+		Long:  "Get information about the document. You can either put the documentId or the sharedId as flag",
+		Args:  cobra.MinimumNArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if documentGetId != "" {
+				return documentGetById(cfg.serverUrl, cfg.apiKey, outline.DocumentID(documentGetId))
+			}
+			if documentGetShareId != "" {
+				return documentGetByShareId(cfg.serverUrl, cfg.apiKey, outline.DocumentShareID(documentGetShareId))
+			}
+			return fmt.Errorf("you should either provide --id or --sharedId")
+		},
+	}
+	documentGetCmd.Flags().StringVar(&documentGetId, "id", "", "The id of the document")
+	documentGetCmd.Flags().StringVar(&documentGetShareId, "shareId", "", "The sharedId of the document")
```

Comment:
I think we discussed [here](https://github.com/ioki-mobility/go-outline/discussions/58#discussioncomment-7088642) best practice is to first establish some default behavior if possible then use options to override/modify it. Here there is no nee for two options. You are basically asking trying to make one 'option' 'compulsory' which is confusing. Better make a required argument and use an option to treat the value differently.

Reply:
@StefMa shall I make the change?

Reply:
Sorry for not coming back earlier to this. I had a lot to do quite recently 😔.
However, I guess it make sense that you take over as Iam not 100% sure how we want to achieve this ☺. Yeah, feel free to make the changes. I will have a look afterward 👍 thank you!

Reply:
okay then!